.prettierrc.json

```diff
@@ -0,0 +1,4 @@
+{
+    "singleQuote": true,
+    "trailingComma": "none"
```

Comment:
I'd suggest turning this on, since it's pretty handy when copy/pasting lines.

Reply:
That's my preference as well. I'm wanting to avoid changing the config however so this PR is focused on simply enforcing the rules as they are.

Once that's done we can open new PRs to update the config so that it can be discussed separately.